content/en/docs/setup/best-practices/certificates.md

```diff
@@ -127,6 +137,32 @@ Same considerations apply for the service account key pair:
 |  sa.key                      |                             | kube-controller-manager | --service-account-private-key-file   |
 |                              | sa.pub                      | kube-apiserver          | --service-account-key-file           |
 
+The following example illustrates the files from the previous tables you need to provide if you are generating all of your own keys and certificates:
```

Comment:
nit: possibly reword line 140
The following example lists the file paths for the generated keys and certificates [from the previous table (Certificates Path Table)].

Reply:
I assume you want me to replace "Certificates Path Table" with a link to the beginning of the section holding the two tables. Done.